### ISSUE TITLE
[SUGGESTION] Handrail doors

### ISSUE THREAD

**Issue by Scwestbrook** [KyleLin921021/EkiMod#9]
To go with the handrail, could you add doors of the same height and size for the station that slide into the nearest handrail when powered with redstone?

*Wow. I've been commenting on your mods quite a bit :)*

**Comment by ChAoSUnItY** [comment]
I think you're talking about like this one?
![image](https://user-images.githubusercontent.com/43753315/102842358-c1ce0700-4441-11eb-8560-73e94eb3fd54.png)

**Comment by Scwestbrook** [comment]
Exactly! Actually, I visited that exact station, or a very similar one, which is where I got the idea.
